ts/util/table-builder.ts

```diff
@@ -124,9 +124,8 @@ export class TableBuilder<ParentType, ChildType> {
     this.addFilterControls(topControlsContainer);
     this.addPagingControls(topControlsContainer);
     this.container.appendChild(topControlsContainer);
-
     const tableContainer = document.createElement("div");
-    tableContainer.className = "mt-4 overflow-auto";
+    tableContainer.className = "mt-4 overflow-x-auto overflow-y-hidden";
```

Comment:
Can revert this to `overflow-auto` to make it more obvious if we ever have another issue like the one with the tooltips clipping vs. creating vertical scrollbars